cmd/kube-controller-manager/app/certificates.go

```diff
@@ -224,3 +223,80 @@ func startRootCACertificatePublisherController(ctx context.Context, controllerCo
 	go sac.Run(ctx, 1)
 	return nil, true, nil
 }
+
+func newKubeAPIServerSignerClusterTrustBundledPublisherDescriptor() *ControllerDescriptor {
+	return &ControllerDescriptor{
+		name:     names.KubeAPIServerClusterTrustBundlePublisherController,
+		aliases:  []string{"kube-apiserver-serving-clustertrustbundle-publisher"},
+		initFunc: newKubeAPIServerSignerClusterTrustBundledPublisherController,
+	}
```

Comment:
Until we make CTB beta:

```suggestion
		initFunc: newKubeAPIServerSignerClusterTrustBundledPublisherController,
		isDisabledByDefault: true,
	}
```